### DISCUSSION TITLE
Can I use DuckDB for this scenario?

### DISCUSSION THREAD

**Discussion by sharadkr14** [duckdb/duckdb#7224] [Q&A]
Can we use Duck DB for the following scenario / use case for "small data" analytics? Put DuckDB in a Kubernetes container, load data and allow multiple users to analyze data concurrently? Some users may be connecting to it via Tableau dashboard, some direct SQL from a workbench, some via Python notebook. This will be for read only queries. So essentially as a "replacement" for Big Query or Snowflake?

**Comment by djouallah** [comment]
today duckdb don't have SQL Endpoint, you can use odbc but you need a direct access to the file, so not sure how this scenario is feasible.
